_docs/ios/1.168.0.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: page
+title: 1.168.0
+permalink: /docs/ios/1.168.0/
+parent: iOS
+nav_order: -168
+release_date: 27.03.2023
+---
+
+## Issue with starting calibration after enabling the "Bluetooth only" mode 
+We fixed the issue that prevent user from starting calibration process just after "Bluetooth only" mode was enabled. Workaround was to go back to home screen, wait until lock connect via BLE and then calibration process was working corectly.
+
+## Position of loading animation on Apple Watch
+We fixed the issue on Apple Watch companion app, that after presing open or close button, shows progess indicator in wrong position.\
```

Comment:
is this slash on end required here?

Reply:
Yes, this is a line break and by doing this I prevent situation where image could appears in side the text.